### PR TITLE
Сreating a color_shade_gen that will generate images with different shades of color to test the check_text function on different background colors

### PR DESCRIPTION
**Pull request by Giovaaanniii**
I wrote a function that changes the color of the image, and also wrote a small test to check if the tesseract detects text in the final image

### CODE REVIEW

File: tests/find/gen.py [hunk 1]
```diff
@@ -6,14 +8,82 @@ def crop_image(img, w, h):
...
     return img
+
+
+def color_shade_gen(rgb: tuple, step: int, size: tuple = (1920, 1080), crop=False):
+    """Function generates images with different shades of color
+
+    Args:
+        rgb (list): List of color values in the rgb color model
+        step (int): Step that determines the frequency of creating shades of color
+        size (tuple): Specifying the size of the image to be created
```
Comment (vsysoev) [inline tests/find/gen.py:45]: size is also optional

File: tests/find/gen.py [hunk 2]
```diff
@@ -6,14 +8,82 @@ def crop_image(img, w, h):
...
+            elif item[2] > 200 and item[0] < 80 and item[1] < 80:
+                new_image_data.append((0, 0, color)) # Blue color
+        new_img = Image.new('RGB', img.size)
+        new_img.putdata(new_image_data)
+        new_img.save(f"tests/find/color_shades/color{color}.png", 'PNG')
+
+color_shade_gen((210, 0, 0), 5, (1280,1280))
+
+
+        # if item[0]>200 and item[1]<80 and item[2]<80:
```
Comment (vsysoev) [inline tests/find/gen.py:73]: These comments should be removed

**Review by vsysoev — CHANGES_REQUESTED** [review]
Please do some cleanup

**Comment by vsysoev** [comment]
Goal: Define limit of function check_text. Define constraints for tesseract recognition functions

**Comment by Giovaaanniii** [comment]
incorrectly created branch

**Pull request closed without merging**